### DISCUSSION TITLE
how to use the background image?

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#2251] [Help] [answered]
Hello,

I cannot find any info in your documentations regarding how to use the background image.

I would like to put a large image in the background of the layout. Is it even possible?

Thanks a lot :)

**Comment by kevmodrome** [comment, marked as answer]
Hi @oswaldB you would need to manually add one using the custom CSS feature.

**Reply by joebudi** [reply]
```
position: fixed;
top: 0;
right: 0;
bottom: 0;
left:0;
z-index: -1;
```


Set the height to whatever you want. That should do it. You might need to play around a bit.

**Comment by oswaldB** [comment]
Thank you ;)

**Reply by joebudi** [reply]
100vh usually does the trick from memory!

It also depends on the layout. The following could also be possible depending on the layout:

```
position: absolute;
top: 0;
right: 0;
bottom: 0;
left:0;
z-index: -1;
```
